datafusion/sqllogictest/test_files/array_query.slt

```diff
@@ -93,11 +93,11 @@ query ??I
 SELECT * FROM data WHERE null = column2;
 ----
 
-query ??I
+query ??I rowsort
```

Comment:
wondering should we also fix the test below?

Reply:
The one right below has a single output row, so it doesn't need to be sorted.